src/components/Validator.js

```diff
@@ -36,7 +36,7 @@ export var Validator = {
     _each(component.validators, (name) => {
       if (this.validators.hasOwnProperty(name)) {
         let validator = this.validators[name];
-        if (component.component.multiple && _isArray(value)) {
+        if ((component.component.multiple && _isArray(value)) && !(component.component.type === 'select' && validator.key === 'validate.required')) {
```

Comment:
I would prefer we do this in an OOP way.  Where there is a method within ```BaseComponent``` called ```isMultiple``` that looks something like this.

```
isMultiple(value) {
  return this.component.multiple && _isArray(value);
}
```

And then in the ```SelectComponent``` you would do something like...

```
isMultiple(value) {
  return false;
}
```

This could then be changed to...

```
if (component.isMultiple(value)) {
```

Which is much cleaner IMO.